pandas/io/json.py

```diff
@@ -197,7 +204,29 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
 
     Returns
     -------
-    result : Series or DataFrame
+    result : Series or DataFrame, depending on the value of ``typ``.
+
+    Examples
+    --------
+
+    >>> df = pd.DataFrame([['a', 'b'], ['c', 'd']],
+                              index=['row 1', 'row 2'],
+                              columns=['col 1', 'col 2'])
+    >>> print df
+          col 1 col 2
+    row 1     a     b
+    row 2     c     d
+    >>> for orient in ['split', 'records', 'index']:
+            str = df.to_json(orient=orient)
+            print "'{}': '{}'".format(orient, str)
+            pd.read_json(str, orient=orient)
```

Comment:
I would just use separate lines instead of the for loop, I personally think this is going to be clearer for the reader

I mean like 

```
>>> df.to_json(orient='split')
.. output ..

>>> df.to_json(orient='records')
.. output ..

....
```